<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorManagement</class>
 <widget class="QDialog" name="ColorManagement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1035</width>
    <height>662</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>480</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="corAzulLowerH">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>40</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="corAzulLowerS">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>40</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="corAzulLowerV">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>40</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="corAzulUpperH">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>70</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="corAzulUpperS">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>70</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="corAzulUpperV">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>70</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="corAzulLabel">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>10</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Azul:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="corLaranjaUpperV">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>270</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="corLaranjaLabel">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>210</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Laranja:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="corLaranjaUpperH">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>270</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="corLaranjaLowerV">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>240</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="corLaranjaLowerS">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>240</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="corLaranjaLowerH">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>240</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="corLaranjaUpperS">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>270</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonAzul">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>60</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Calibrar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonLaranja">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>250</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Calibrar</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="cor1UpperV">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>380</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="cor1LowerS">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>350</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="cor1LowerH">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>350</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCor1">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>360</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Calibrar</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="cor1UpperH">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>380</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="cor1Label">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>320</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Cor 1:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="cor1LowerV">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>350</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="cor1UpperS">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>380</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="cor2LowerV">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>460</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="cor2UpperV">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>490</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="cor2UpperH">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>490</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="cor2LowerS">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>460</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCor2">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>470</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Calibrar</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="cor2LowerH">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>460</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="cor2UpperS">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>490</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="cor2Label">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>430</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Cor 2:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="cor3LowerV">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>560</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="cor3Label">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>530</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Cor 3:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="cor3UpperV">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>590</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="cor3UpperS">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>590</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="cor3UpperH">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>590</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="cor3LowerS">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>560</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="cor3LowerH">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>560</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCor3">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>570</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Calibrar</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="corAmareloLowerV">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>130</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="corAmareloUpperV">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>160</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="corAmareloUpperS">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>160</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="corAmareloUpperH">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>160</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="corAmareloLowerS">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>130</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="corAmareloLowerH">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>130</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonAmarelo">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>150</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Calibrar</string>
   </property>
  </widget>
  <widget class="QLabel" name="corAmareloLabel">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>100</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Amarelo:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="mostrarFiltroCheckBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>510</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Mostrar filtro</string>
   </property>
  </widget>
  <widget class="QPushButton" name="salvar">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>510</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Salvar novas configurações</string>
   </property>
  </widget>
  <widget class="QPushButton" name="reset">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>550</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Voltar as configurações antigas</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
